server/aesdsocket.c

```diff
@@ -110,6 +118,28 @@ void *connection_thread_func(void* thread_param) {
 		memset(recv_buf, 0, RECV_BUF_LEN);
 	}
 
+#ifdef USE_AESD_CHAR_DEVICE
+	if(strncmp(dyn_recv_buf, g_iocseek_str, g_ioc_len) == 0) {
+		seekto.write_cmd = dyn_recv_buf[g_ioc_x] - '0';
+		seekto.write_cmd_offset = dyn_recv_buf[g_ioc_y] - '0';
```

Comment:
What does this assume about the size of the seek write command and offset values?